lib/offline.js

```diff
@@ -119,19 +131,23 @@ module.exports = function(config) {
         var data = fs.readFileSync(filepath);
         totalGzipSize += gzipSize.sync(data);
         var hash = getHash(data);
-        process.stdout.write(chalk.green.bold('✓ ') + 'Caching ' + filepath + ' (' + prettyBytes(size) + ')\n');
+        var path = getPathName(filepath, rootDir);
+
+        process.stdout.write(chalk.green.bold('✓ ') + 'Caching ' + filepath + ' at path ' + path + ' (' + prettyBytes(size) + ')\n');
```

Comment:
I would only add a reference to the path for files that match a *directoryIndexes* value (and whose paths are thus being modified); otherwise the reference doesn't stand out, and the output is cluttered by largely-redundant references to paths that resemble their files, f.e.:

```
> oghliner offline --directory-indexes index.html dist/
Offlining dist/ to dist/offline-worker.js…
✓ Caching dist/CNAME at path ./CNAME (13 B)
✓ Caching dist/css/index.css at path ./css/index.css (378 B)
✓ Caching dist/favicon/android-chrome-144x144.png at path ./favicon/android-chrome-144x144.png (2.65 kB)
…
✓ Caching dist/index.css at path ./index.css (223 B)
✓ Caching dist/index.html at path ./ (3.04 kB)
✓ Caching dist/index.js at path ./index.js (4.29 kB)
```